models/signed-resource.js

```diff
@@ -27,6 +26,11 @@ export default class SignedResource {
         ?blockchainStatus mu:uuid ?blockchainStatusUuid.
         ?signatory mu:uuid ?signatoryUuid.
         
+        OPTIONAL {
+          { ?uri sign:text ?html. }
+          UNION
+          { ?uri prov:generated ?file. }
+        }
```

Comment:
I'm wondering if the `OPTIONAL` operator is necessary here, as it always needs to contain at least one of the two predicates right?

Reply:
You're right that it's not needed and it mimics the previous behaviour, which required the `sign:text` triple to exist. I've changed that now.